### ISSUE TITLE
Bump mailparser to fix CVE-2020-7769 for inbound-mail-parser 

### ISSUE THREAD

**Issue by atkeson** [sendgrid/sendgrid-nodejs#1423]
mailparser `2.3.4` (from +4 years ago) relies on an old version of nodemailer which is vulnerable to [CVE-2020-7769](https://nvd.nist.gov/vuln/detail/CVE-2020-7769) NIST base score of 9.8. New versions of mailparser use an updated version of nodemailer which paches this vuln. Can we please update the mailparser version?

**Comment by davidyell** [comment]
Yeah, this needs to be fixed, this is reporting as a critical issue in `npm audit`
